test/service_tests/multi_media_pick_service_test.dart

```diff
@@ -41,10 +41,6 @@ void main() {
       when(
         mockImageCropper.cropImage(
           sourcePath: "test",
-          aspectRatioPresets: [
-            CropAspectRatioPreset.square,

```

Comment:
1. You have removed testing features.
2. We need to codebase to be stable and this defeats the purpose
3. Please make the tests valid